images/win/scripts/Installers/Configure-PowerShell.ps1

```diff
@@ -0,0 +1,29 @@
+################################################################################
+##  File:  Configure-Powershell.ps1
+##  Desc:  Manage PowerShell configuration
+################################################################################
+
+#region System
+Write-Host "Setup PowerShellGet"
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+
+# Specifies the installation policy
+Set-PSRepository -InstallationPolicy Trusted -Name PSGallery
+#endregion
+
+#region User (current user, image generation only)
+if( -not(Test-Path $profile)) {
+    New-Item $profile -ItemType File -Force
+  }
```

Comment:
Please bring code into line with style guide (spaces in line 15 and indentation in line 17)

Reply:
thanks, much better indeed. I'll submit a patch